app/controllers/api/base_controller.rb

```diff
@@ -276,7 +276,7 @@ def parent_permission(child_permission)
       case child_permission.to_s
       when 'create', 'destroy'
         'edit'
-      when 'edit', 'view'
+      when 'edit', 'view', 'import_puppetclasses'
```

Comment:
Define `parent_permission` on `EnvironmentsController` if it's really needed - can you elaborate what this is for? I understand the other change in the PR